### ISSUE TITLE
Double Declaration

### ISSUE THREAD

**Issue by rahrah** [xroche/httrack#63]
Hi,

On compiling httrack-3.48.21 under Linux (gcc 4.5.2), I get this error:

```
In file included from htsname.c:39:0:
md5.h:47:27: error: redefinition of typedef 'MD5CTX'
md5.h:28:3: note: previous declaration of 'MD5CTX' was here
```
Patching with this:

```
--- src/md5.h.org       2015-03-15 16:13:31.000000000 +0000
+++ src/md5.h   2015-03-19 15:12:18.000000000 +0000
@@ -44,6 +44,6 @@
 /*
 * This is needed to make RSAREF happy on some MS-DOS compilers.
 */
-typedef struct MD5Context MD5CTX;
+/* typedef struct MD5Context MD5CTX; */
 
 #endif /* !MD5_H */
```
works for me.

This change was introduced, it seems in this commit:

```
commit ebd4d34b7b91ccee5e3fbaa587fa320acf3ce7ac
Author: Xavier Roche <xroche@users.noreply.github.com>
Date:   Sun Jul 6 08:17:09 2014 +0000
```

Thanks,

===Rich

**Comment by rahrah** [comment]
Hello,

Somehow 
```
commit 6e2167c1a128f2d9c1e25c4b2e8187f064006d72 
```
has been overwritten.

It's now 
```
commit f88979989014e3c45e550fa10808ee1ddad163cf
```
It has the same tree and diff, but the wrong commit message.

It was and should be:

```
Fixed duplicate declaration of MD5CTX (rahrah) (closes  #63)
```
Now:

```
Fixed broken symlink with changed --docdir (rahrah) (closes #65)
```
===Rich

**Comment by xroche** [comment]
Sorry for the mess, this should be cleaned now!
